finance/expenses/Portfile

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/manojkarthick/expenses 0.2.2 v
+categories          finance
+maintainers         {gmail.com:smanojkarthick @manojkarthick} \
+                    openmaintainer
+license             MIT
+
+description         Simple, Interactive, command line Expense logger.
```

Comment:
nitpick: I'm not sure whether there's a consensus on this, but generally the `description` doesn't have a full stop whilst the `long_description` does.